packages/shorebird_cli/test/src/platform/apple_test.dart

```diff
@@ -92,6 +92,39 @@ To add macOS, run "flutter create . --platforms macos"''');
         ).thenReturn(projectRoot);
       });
 
+      group('supplement artifact', () {
+        test('copySupplementFilesNextToSnapshots', () {
+          final names = [
+            'App.class_table.json',
+            'App.field_table.json',
+            'App.ct.link',
+            'App.ft.link',
+          ];
+
+          void createFiles(Directory dir) {
+            for (final name in names) {
+              File(p.join(dir.path, name)).createSync();
+            }
+          }
+
+          final releaseSupplementDir = Directory.systemTemp.createTempSync();
+          final patchSupplementDir = Directory.systemTemp.createTempSync();
+          createFiles(releaseSupplementDir);
+          createFiles(patchSupplementDir);
+
+          final releaseSnapshotDir = Directory.systemTemp.createTempSync();
+          final patchSnapshotDir = Directory.systemTemp.createTempSync();
+          apple.copySupplementFilesToSnapshotDirs(
+            releaseSupplementDir: releaseSupplementDir,
+            releaseSnapshotDir: releaseSnapshotDir,
+            patchSupplementDir: patchSupplementDir,
+            patchSnapshotDir: patchSnapshotDir,
+          );
+          expect(Directory(releaseSnapshotDir.path).listSync(), hasLength(4));
```

Comment:
We should probably also have a test that includes a subset of the supplementary artifacts to ensure the copy handles that case correctly.

Reply:
done.